src/org/zaproxy/zap/extension/pscanrules/TestInfoPrivateAddressDisclosure.java

```diff
@@ -108,7 +108,7 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
         
         while (matcher.find()) {
             String privateIp = matcher.group();
-            if (getLevel() != AlertThreshold.LOW && privateIp.equalsIgnoreCase(host)) {
+            if (getLevel() != AlertThreshold.LOW && privateIp.toLowerCase().contains(host)) {

```

Comment:
It could be used `matcher.group(1)` for this check, which returns just the private IP. Otherwise the `host` should be lowered case (and used `Locale.ROOT`), worth adding a test for mixed case?